### PR TITLE
Feature/bc 167 layouts

### PR DESCRIPTION
**Pull request by pkleybolte**
# Short Description
I created a wireframe that is supposed to be the basic of future pages.
The wireframe contains the breadcrump and title to keep a consistent layout.
The actual page content should go into the default slot.

## Links to Ticket and related Pull-Requests
https://ticketsystem.hpi-schul-cloud.org/browse/BC-167

## Checklist before merging

- [x] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] PO: Any deviation from requirements was agreed with Product-Owner / ticket author / support-team

> Notice: Please keep this Pull-Request as a Draft (or add WIP label), until it is ready to be reviewed

### CODE REVIEW

**Review by murat-merdoglu-hpi — DISMISSED** [review]

File: locale/es.json [hunk 1]
```diff
@@ -199,6 +199,10 @@
 	"global.sidebar.tasksUngraded": "Sin auditar",
 	"global.sidebar.teacher": "Profesor",
 	"global.sidebar.teams": "Equipos",
+	"global.topbar.actions.contactAdmin": "Contacta a al admin de tu escuela",
```
Comment (odalys-dataport) [inline locale/es.json:202]: This should be only "al"

File: locale/de.json [hunk 2]
```diff
@@ -200,6 +200,10 @@
 	"global.sidebar.tasks.tasksCurrent": "Aktuelle Aufgaben",
 	"global.sidebar.teacher": "Lehrkräfte",
 	"global.sidebar.teams": "Teams",
+	"global.topbar.actions.contactAdmin": "Admin deiner Schule kontaktieren",
+	"global.topbar.actions.contactSupport": "Wunsch oder Problem senden",
+	"global.topbar.actions.help": "Hilfebereich",
+	"global.topbar.actions.training": "Fortbildungen",
```
Comment (odalys-dataport) [inline locale/de.json:206]: Reminder to compare with legacy client translations

File: src/components/molecules/DefaultWireframe.unit.js [hunk 3]
```diff
@@ -0,0 +1,101 @@
+import { mount } from "@vue/test-utils";
```
Comment (odalys-dataport) [inline src/components/molecules/DefaultWireframe.unit.js:1]: `DefaultWireframe` component should go into templates (folder structure)

File: src/components/molecules/DefaultWireframe.unit.js [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+	it("shows title", () => {
+		const wrapper = mount(DefaultWireframe, {
+			...createComponentMocks({}),
+			propsData: { headline: "dummy titel" },
+		});
+
+		const h1 = wrapper.find("h1");
+		expect(h1.text()).toBe("dummy titel");
+	});
+	it("shows breadcrumps", () => {
```
Comment (odalys-dataport) [inline src/components/molecules/DefaultWireframe.unit.js:14]: *breadcrumbs

File: src/components/molecules/DefaultWireframe.vue [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+				'container-full-width': fullWidth,
+			}"
+			class="main-content"
+		>
+			<slot></slot>
+		</v-container>
+	</v-container>
+</template>
+<script lang="ts">
+import vCustomBreadcrumbs from "@components/molecules/vCustomBreadcrumbs.vue";
```
Comment (odalys-dataport) [inline src/components/molecules/DefaultWireframe.vue:24]: `vCustomBreadcrumbs` probably should be an atom

File: src/components/molecules/DefaultWireframe.vue [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+
+export type Breadcrump = {
```
Comment (odalys-dataport) [inline src/components/molecules/DefaultWireframe.vue:26]: *Breadcrumb

File: src/components/molecules/DefaultWireframe.vue [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+	},
+};
+</script>
+<style lang="scss" scoped>
+@import "@styles";
+
+.container.wireframe-container {
+	padding: 0 var(--space-lg); // Desktop
+	// padding: 0 36px;
+	// padding: 0 36px;
```
Comment (odalys-dataport) [inline src/components/molecules/DefaultWireframe.vue:62]: can this be removed?

File: src/pages/administration/school-settings.vue [hunk 8]
```diff
@@ -1,71 +1,69 @@
 <template>
-	<v-container fluid>
-		<v-custom-breadcrumbs :breadcrumbs="breadcrumbs"></v-custom-breadcrumbs>
-		<v-container class="container-max-width">
-			<h1 class="text-h3 page-title">
-				{{ $t("pages.administration.school.index.title") }}
-			</h1>
-			<v-alert v-if="schoolError" light prominent text type="error">
-				{{ $t("pages.administration.school.index.error") }}
+	<default-wireframe
```
Comment (odalys-dataport) [inline src/pages/administration/school-settings.vue:2]: Idea: Default loading state for full page

File: src/pages/administration/students/index.vue [hunk 9]
```diff
@@ -1,6 +1,10 @@
 <!-- eslint-disable max-lines -->
 <template>
-	<section class="section">
+	<default-wireframe
+		:headline="$t('pages.administration.students.index.title')"
+		:breadcrumbs="breadcrumbs"
+		:full-width="true"
```
Comment (odalys-dataport) [inline src/pages/administration/students/index.vue:6]: If it feels more comfortable to be explicit here, we could make full-width required as well

**Review by odalys-dataport — CHANGES_REQUESTED** [review]

**Review by odalys-dataport — DISMISSED** [review]

**Review by odalys-dataport — APPROVED** [review]

**Review by murat-merdoglu-hpi — APPROVED** [review]

**Pull request merged**